README.md

```diff
@@ -31,6 +31,7 @@ will replace [Ed-Fi ODS/API 7.x](https://techdocs.ed-fi.org/x/UQSUCg).
 - [Running in Local Context](./docs/RUNNING-LOCALLY.md)
 - [Configuration](./docs/CONFIGURATION.md)
 - [Docker](./docs/DOCKER.md)
+- [Reference-Validation](./docs/REFERENCE-VALIDATION.md)
```

Comment:
Suggestion: "Removing reference validation" so that the reader has a better idea of what to expect with this article.